### PR TITLE
Particles wrapper

### PR DESCRIPTION
**Pull request by itziakos**
This PR provides an adapter for a vtk.PolyData dataset to the Particles CUDS container api.

### CODE REVIEW

**Comment by itziakos** [comment]
This PR should be merged after #59

File: simphony_mayavi/core/cuba_data.py [hunk 1]
```diff
@@ -236,3 +244,48 @@ def _array_value(self, name, values):
...
+                data.get_array_name(array_id))
+            for array_id in range(data.number_of_arrays)}
+        message = "vtk object arrays are not the same size: {}"
+        raise ValueError(message.format(info))
+    elif len(lengths) == 0:
+        # empty vtk object
+        return
+    else:
+        # fix masked arrays
+        length = next(iter(lengths))
```
Comment (dpinte) [inline simphony_mayavi/core/cuba_data.py:284]: `next(iter(lenghts))` to get the item 1 of lengths is not fun to read. I'm not sure what to suggest there.`lenghts.pop()` ?

**Comment by dpinte** [comment]
@itziakos quick review, LGTM with one minor comment

File: simphony_mayavi/core/cuba_data.py (hunk 1 above)
Comment (itziakos) [inline simphony_mayavi/core/cuba_data.py:284]: `lengths` is a set so `pop` is not available. alternative one needs to do `tuple(lengths)[0]` which I always find uglier.

**Pull request merged**